### DISCUSSION TITLE
MissionControl 0.7.0

### DISCUSSION THREAD

**Discussion by Anish5150** [ndeadly/MissionControl#492] [Q&A]
Will MissionControl 0.7.0 work on Atmosphère:1.3.2 on firmware 14.1.2. I am new to switch hacking scene that's why I am asking. Thanks

**Comment by ndeadly** [comment]
Yes, should do. 14.1.2 was only a minor update and didn't even require an update to Atmosphere. No Atmosphere update = no Mission Control update required.
